regression/contracts-dfcc/chain.sh

```diff
@@ -59,4 +59,4 @@ elif echo $args_inst | grep -q -- "--dump-c" ; then
   rm "${name}${dfcc_suffix}-mod.c"
 fi
 $goto_instrument --show-goto-functions "${name}${dfcc_suffix}-mod.gb"
-$cbmc "${name}${dfcc_suffix}-mod.gb" ${args_cbmc}
+$cbmc --no-standard-checks "${name}${dfcc_suffix}-mod.gb" ${args_cbmc}
```

Comment:
Please don't do this. It will be very though for anyone to work out what the tests actually do execute when one fails.

Reply:
Hi @kroening, just for clarification:

We were hoping to use this approach as a stop-gap to allow us to get the regression test suite in a way that is all-green again, so that we could merge this in time, seeing as this is the main ticket that's blocking a v6 release.

But the aim was to get this in temporarily, and as soon our v6 release is unblocked, reopen #8006, rebase it and then move to a model where the extra flag is being passed on an individual basis to tests (to localise the information about the test on the individual `.desc` file).

The original approach (of getting all the `.desc` readjusted) was found to be time consuming, and we wanted to at least unblock the release before we embark on that.

Is that something that works for you? If not, is there a particular proposal you had mind?

Reply:
It's ok to use `--no-standard-checks` on most of the tests; my suggestion is to add it to the `.desc` file, where everyone will see it, instead of burying it inside a script.

Reply:
I agree that the place for the `--no-standard-checks` flag is the `.desc` file of each failing test.
Unfortunately there are about `300` failing tests, so the idea was to avoid having a PR with code changes in the middle of `300` repetitive test fixes, but instead to temporarily add `--no-standard-checks` to the scripts to pass CI and them fix all the failing test on a subsequent PR aimed solely at fixing the regression tests.

@kroening I hope that this clarifies why the `--no-standard-checks` flag has been added on the runner script at the moment.